src/main/java/no/nav/pto/veilarbportefolje/fargekategori/FargekategoriController.java

```diff
@@ -87,6 +91,43 @@ public ResponseEntity<FargekategoriResponse> oppdaterFargekategoriForBruker(@Req
         }
     }
 
+    @PutMapping("/fargekategorier")
+    public ResponseEntity batchoppdaterFargekategoriForBruker(@RequestBody BatchoppdaterFargekategoriRequest request) {
```

Comment:
Er det nødvendig å ha to forskjellige funksjoner for oppdatering? For å gjøre det enkelt så så vi for oss å sende inn en liste med ett element når det kun er én bruker man endrer!

Reply:
Nei, dette var bare tilfell eman ville lage ting ferdig for frontend der man er nå (for 1 bruker). Tanken er å slette det andre endepunktet 👍